libgnucash/engine/Account.cpp

```diff
@@ -248,6 +249,78 @@ mark_account (Account *acc)
     qof_instance_set_dirty(&acc->inst);
 }
 
+Account*
+gnc_account_from_tuple (const AccountTuple& t, QofBook* book)
```

Comment:
I suppose this slipped in  by accident ?

Reply:
Yup, sorry, it belongs to another branch. I thought I had a clean tree.